api/proto/buf.lock

```diff
@@ -0,0 +1,7 @@
+# Generated by buf. DO NOT EDIT.
```

Comment:
Add this file to https://github.com/apache/skywalking-banyandb/blob/808fc8faec769de5f7b915eba7998f30a6de5bd3/.licenserc.yaml#L63